src/main/java/utils/TestUtils.java

```diff
@@ -11,7 +11,7 @@ public static String randomString(int numberOfCharacters) {
         return randomString(numberOfCharacters, ASCII_LETTERS);
     }
 
-    private static String randomString(int numberCount, CharacterPredicate characterPredicate) {
+    public static String randomString(int numberCount, CharacterPredicate characterPredicate) {
```

Comment:
Why it is changed to public. I don't see that it was used in any other classes than this one. 
So it should stay private.